examples/js/loaders/OBJLoader.js

```diff
@@ -711,151 +711,179 @@ THREE.OBJLoader = ( function () {
 			var container = new THREE.Group();
 			container.materialLibraries = [].concat( state.materialLibraries );
 
-			for ( var i = 0, l = state.objects.length; i < l; i ++ ) {
+			if ( state.objects.length === 1 && state.objects[ 0 ].geometry.vertices.length === 0 ) {
```

Comment:
Is there a way to make this a bit more clear? Maybe a `hasFaces` boolean?

Reply:
I've updated the PR with a new commit. I've used `hasPrimitives` since lines or points can be defined, too.